### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by cbrenner04**
@michaelwehrley I updated the README from your branch msw_94587836_update_install_readme. I was having trouble resolving merge conflicts so just opened a new branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,20 +159,22 @@ npm run appium:stop
...
-The Travis configuration file, .travis.yml, will cause Travis to run our tests when code is pushed to GitHub.
+The Travis configuration file, .travis.yml, will cause Travis to run our tests
+when code is pushed to GitHub.
 
 ## Additional information
 
 ### Contact
 
-For more information on **E.D. Navigator App** please check out http://www.cbits.northwestern.edu/
+For more information on **E.D. Navigator App** please check out [CBITs](http://www.cbits.northwestern.edu/)
```
Comment (ericcf) [inline README.md]: I don't see anything about this app on that website.

File: README.md [hunk 2]
```diff
@@ -18,26 +20,72 @@ npm run build:prepare
...
+npm run build:update_content
 ```
 
 Run the project on the simulator (prepare, compile and install):
+
+```
+npm run android:simulator
+```
+
+#### Installing apk on Android Device
```
Comment (ericcf) [inline README.md]: Rather than having project specific instructions, can these be put in our guide and linked to?

**Comment by michaelwehrley** [comment]
:eyeglasses: good to me.  Thanks.

File: README.md (hunk 2 above)
Comment (michaelwehrley) [inline README.md]: That is a good idea.

Comment (cbrenner04) [inline README.md]: I can work on that. @ericcf @michaelwehrley for the guides, I would likely have to extract the commands that are being run in the app specific commands that wehrley configured, correct? For example `android:simulator` is actually running 
```
rm -rf www && mkdir -p www/js www/css && cp src/index.html www/ && cp -r src/js/* www/js/ && cp -r src/css/* www/css/ && cp ./node_modules/angular/angular.min.js www/js/ && cp ./node_modules/angular-route/angular-route.min.js www/js/ && cp ./node_modules/angular-resource/angular-resource.min.js www/js/ && mkdir www/partials/ && cp -r src/partials/* www/partials/ && mkdir -p www/vendor/css/ && cp -r src/vendor/css/* www/vendor/css/ && mkdir www/images/ && cp -r src/images/* www/images/ && mkdir www/videos/ && cp -r src/videos/* www/videos/ && mkdir -p www/vendor/css/font-awesome/fonts && cp -r ./node_modules/font-awesome/fonts/* www/vendor/css/font-awesome/fonts && mkdir -p www/vendor/css/font-awesome/css && cp ./node_modules/font-awesome/css/font-awesome.min.css www/vendor/css/font-awesome/css && ./node_modules/.bin/cordova build android && ./node_modules/.bin/cordova run android
```
Or are these commands standard?

Comment (michaelwehrley) [inline README.md]: Those are specific to each app.  What I think @ericcf means, is that to put these commands in guides and then link to them from this readme: https://github.com/cbitstech/ed_navigator_app/blob/cjb_update_readme/README.md#installing-apk-on-android-device

File: README.md (hunk 1 above)
Comment (cbrenner04) [inline README.md]: @michaelwehrley what do you want to do with this? This was originally in your README. I was thinking about just changing this to "For more information about CBITs please check out...

File: README.md (hunk 2 above)
Comment (cbrenner04) [inline README.md]: ok. I'll take care of that. That makes way more sense. Thanks @michaelwehrley

File: README.md (hunk 1 above)
Comment (michaelwehrley) [inline README.md]: lol.  We can just delete it I guess.  I like that we link there, but the site url doesn't persist after you click on Research > Projects.  If @ericcf  doesn't want it, then that is fine - we can remove it.

Comment (cbrenner04) [inline README.md]: ok. done

Comment (michaelwehrley) [inline README.md]: What about "For more... on **CBITS** please check out..."?

Comment (cbrenner04) [inline README.md]: @michaelwehrley see my comments above... which do you want? delete or leave it with the update?

Comment (michaelwehrley) [inline README.md]: Let's just delete it.  :scissors: Thanks.

**Comment by cbrenner04** [comment]
@michaelwehrley @ericcf ok. I made the changes. The link to the guides will be broken until that is merged.

**Pull request merged**